R/get_phylopic.R

```diff
@@ -31,36 +36,50 @@ get_phylopic <- function(uuid = NULL, format = "vector") {
   if (!is.character(uuid)) {
     stop("`uuid` is not of class character.")
   }
+  if (as.character(format) %in% c("512", "1024", "1536")) {
+    lifecycle::deprecate_warn("1.1.0",
+                              paste0("get_phylopic(format = '",
+                                     "no longer supports values of ",
+                                     "512, 1024, or 1536')"),
+                              details = paste0("Use the `height` argument ",
+                                               "instead with the `format` ",
+                                               "argument set to \"raster\"."))
+    height <- as.numeric(format)
+    format <- "raster"
+  }
   format <- match.arg(as.character(format),
-                    c("64", "128", "192", "512", "1024", "1536", "twitter",
-                      "vector"))
+                      c("64", "128", "192", "raster", "twitter", "vector"))
```

Comment:
It will be less straightforward to code, but I think you could basically just check for raster, twitter or vector and then if height equals 64, 128, or 192... or just remove this altogether and just create these thumbnails via `rsvg_png`, that way they are always available and you can drop quite a bit of handling here.

Reply:
See my comment above